docs/collect-user-data.md

```diff
@@ -0,0 +1,15 @@
+```
```

Comment:
I thought having an additional copy of the help docs in `docs/` (generated via make) makes it easier to get started (especially non-devs) since you don't need to install the tools to know what they do.